### DISCUSSION TITLE
Adding native Android & iOS implementations.

### DISCUSSION THREAD

**Discussion by mikepattyn** [LuckyPennySoftware/MediatR#782] [Ideas]
Ola, we at Givt, have written Swift and Kotlin versions inspired by your MediatR. 
Do you have any plans for such implementations? Want to colaborate? -> im thinking some advanced marshalling combined with a dotnet maui classlib setup 🤔 

https://github.com/givtnl/Mediater.Swift <-- with SwiftUI example!

**Comment by jbogard** [comment]
No I don't have any plans, I don't do any Swift or Kotlin development. But happy to include links to those!
